tests/dummy/app/locales/ru/translations.js

```diff
@@ -251,7 +251,9 @@ Ember.$.extend(true, translations, {
       'birthday-caption': 'Дата рождения',
       'gender-caption': 'Пол',
       'vip-caption': 'VIP',
-      'karma-caption': 'Карма'
+      'karma-caption': 'Карма',
+      'vname-caption': 'Заполните поле "Имя"',
```

Comment:
v[field-name]-caption is not better key name for validation translation.
Rename it please to something like name-validation-message-caption.
And other similar translations keys too (dont't forget to rename them in model's validation options).